docker-compose.yml

```diff
@@ -259,70 +314,70 @@ services:
     healthcheck:
       test: grpcurl -plaintext localhost:50151 list || exit 1
 
-  testgrpc:
```

Comment:
@udhay-104 do not comment out the entire service, just remove `/entrypoint.sh call...`

Reply:
Done. Can you please check now?